### ISSUE TITLE
Songs are not visible

### ISSUE THREAD

**Issue by Jhalakupadhyay** [Jhalakupadhyay/Sound-Soul#1]
when the app runs it crashes maybe because it doe not find any list of songs from the user external storage

**Comment by ManmeetSingh1012** [comment]
@Jhalakupadhyay  only Gradle files of the project are there in this repo update this repo with the main file

**Comment by Jhalakupadhyay** [comment]
Ohhkk I'll do it by tomorrow for sure

**Comment by Jhalakupadhyay** [comment]
> @Jhalakupadhyay only Gradle files of the project are there in this repo update this repo with the main file

can you now see the main file, actually I am very much new to git so ...

**Comment by ManmeetSingh1012** [comment]
Nope, the main file is still not there, I think your making some mistake while updating the code

**Comment by Jhalakupadhyay** [comment]
> Nope, the main file is still not there, I think you are making some mistake while updating the code

I have fucked up with my app now I am making a new repo of same project please give your contribution to it
